core/scenegraph/src/main/java/edu/cmu/cs/dennisc/scenegraph/Mesh.java

```diff
@@ -112,7 +112,23 @@ protected void updatePlane(Vector3 forward, Vector3 upGuide, Point3 translation)
 
   @Override
   public void transform(AbstractMatrix4x4 trans) {
```

Comment:
Part of me wants to take out the superclass's implementation and the unused unimplemented sibling methods, but this does the job and fits the code structure, such as it is.